hack/build-rpm-release.sh

```diff
@@ -3,9 +3,10 @@
 # This script generates release zips and RPMs into _output/releases.
 # tito and other build dependencies are required on the host. We will
 # be running `hack/build-cross.sh` under the covers, so we transitively
-# consume all of the relevant envars. We also consume:
-#  - BUILD_TESTS: whether or not to build a test RPM, off by default
+# consume all of the relevant envars.
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+os::util::ensure::system_binary_exists tito
+os::util::ensure::system_binary_exists createrepo
```

Comment:
If `createrepo` does not exist, we will not fail, the repodata file for the yum repo that we are mounting to the images will not be created, and eventually our images aare broken.